### ISSUE TITLE
[Bug]: post code starting with 0 shown wrong

### ISSUE THREAD

**Issue by KLIS-IT** [iobroker-community-adapters/ioBroker.tankerkoenig#113]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

In case of station information has a post code starting with 0 it is not shown in station information and data point postCode
![image](https://user-images.githubusercontent.com/20384874/210433657-c1ef2eb7-cc9e-4d57-a66a-8f53800bfded.png)

### What did you expect to happen?

post codes starting with 0 are shown correct (as string not number)

### How to reproduce it (minimal and precise)

add new station with post code starting with 0

### Adapter version

3.3.3

### Environment

```markdown
- Node: v16.15.1
- npm: 8.11.0
- JS-Controller: 4.0.24
- Operating system: Raspbian GNU/Linux 10 (buster)
```

### What browsers are you seeing the problem on?

Chrome

### Screenshots & Log

![image](https://user-images.githubusercontent.com/20384874/210434381-c3925ab7-5b1f-422b-99be-c81eb38352d9.png)

**Comment by xXBJXx** [comment]
The data comes from the server without a preceding 0.
![image](https://user-images.githubusercontent.com/32600934/210438656-491ecf34-46f6-4590-9d24-f406ed50b9cf.png)
the input field is currently set to the type number there is a number with a 0 in front of it not possible I will change the whole thing to string. 
Then you can enter the number with a 0 by hand.

**Comment by xXBJXx** [comment]
should be fixed in version 3.3.4, please note the [Migration to 3.3.4]( https://github.com/iobroker-community-adapters/ioBroker.tankerkoenig/blob/master/docs/guide/migration_3.3.4.md)

**Comment by KLIS-IT** [comment]
Thanks a lot - is working fine with version 3.3.5
